products/godot.md

```diff
@@ -31,7 +31,7 @@ releases:
     release: 2020-01-01
     support: false
     eol: true
-    latest: "3.2.3"
+    latest: "3.2.4"
```

Comment:
Weird - there is no `3.2.4-stable` tag in the https://github.com/godotengine/godot repo, so does the release really exist?

Reply:
I got the pattern: [`3.2.4`](https://downloads.tuxfamily.org/godotengine/3.2.4/) and [`2.1.7`](https://downloads.tuxfamily.org/godotengine/2.1.7/) only contains `beta` and `rc` folders, so they're not stable.

This was the reason why they were not listed as the "latest" versions here.